apps/src/netsim/NetSimLobby.js

```diff
@@ -79,28 +99,48 @@ NetSimLobby.createWithin = function (element, connection) {
   // Create a new NetSimLobby
   var controller = new NetSimLobby(connection);
   element.innerHTML = markup({});
-  controller.initialize();
+  controller.bindElements_();
+  controller.refreshInstanceList_();
   return controller;
 };
 
 /**
- *
- */
-NetSimLobby.prototype.initialize = function () {
-  this.bindElements_();
-  this.refreshInstanceList_();
-};
-
-/**
  * Grab the DOM elements related to this control -once-
  * and bind them to member variables.
  * Also attach method handlers.
  */
 NetSimLobby.prototype.bindElements_ = function () {
+  this.lobbyOpenDiv_ = document.getElementById('netsim_lobby_open');
+  this.lobbyClosedDiv_ = document.getElementById('netsim_lobby_closed');
+
   this.instanceSelector_ = document.getElementById('netsim_instance_select');
   $(this.instanceSelector_).change(this.onInstanceSelectorChange_.bind(this));
 
   this.lobbyList_ = document.getElementById('netsim_lobby_list');
+
+  this.addRouterButton_ = document.getElementById('netsim_lobby_add_router');
+  dom.addClickTouchEvent(this.addRouterButton_,
```

Comment:
Latest on jQuery: yes, we should move to using jQuery for things like this. caveat: mocha tests won't work with jQuery. My proposal: use jQuery here now, and when you go to add tests, there will be an extra step of making mocha pull in jQuery which Brent can help you accomplish.